legal-api/src/legal_api/reports/report.py

```diff
@@ -315,7 +315,7 @@ def _set_completing_party(self, filing):
             self._filing.id, datetime.utcnow(), EntityRole.RoleTypes.completing_party.name
         )
         if completing_party_role:
-            filing["completingParty"] = completing_party_role[0].party.json
+            filing["completingParty"] = completing_party_role[0].json
```

Comment:
Are we sure about this?  The original code went through `party` which is now a `legal_entity` record.  

Can we go through the `related_entity_id` and call `party_json` on the legal entity model?  

![image](https://github.com/bcgov/lear/assets/6685223/fb8f19e5-75e7-40bc-a813-e2071e2cfd3d)

Reply:
Updated